### ISSUE TITLE
Vault API reachable task created malformed url

### ISSUE THREAD

**Issue by etcet** [ansible-community/ansible-vault#88]
I'm using this play:

```
- hosts: ca
  vars:
    vault_backend: file
    vault_cluster_disable: True
    vault_cluster_name: telx
    vault_datacenter: telx
    vault_iface: ens3
  roles:
    - hashicorp-vault
```

During the "Vault API reachable?" task, the URL gets created malformed as seen below in the debug output. It's trying to reach `"\"http\" \"://10.128.18.163\" \":8200/v1/sys/health\"\n"`.

If I instead change the task to construct the url on one line with `"url: "{{ vault_tls_disable | ternary('http', 'https') }}://{{ (vault_address == '0.0.0.0') | ternary('127.0.0.1', vault_address) }}:{{ vault_port }}/v1/sys/health"` it works fine. This is using ansible 2.7.9 on Ubuntu 18.04.

```
FAILED - RETRYING: Vault API reachable? (30 retries left).Result was: {                                                                                                         "attempts": 1,                                                                                                                                                              "changed": false,                                                                                                                                                           "content": "",
    "invocation": {
        "module_args": {
            "attributes": null,
            "backup": null,
            "body": null,
            "body_format": "json",
            "client_cert": null,
            "client_key": null,
            "content": null,
            "creates": null,
            "delimiter": null,
            "dest": null,
            "directory_mode": null,
            "follow": false,
            "follow_redirects": "safe",
            "force": false,
            "force_basic_auth": false,
            "group": null,
            "headers": {
                "Content-Type": "application/json"
            },
            "http_agent": "ansible-httpget",
            "method": "GET",
            "mode": null,
            "owner": null,
            "regexp": null,
            "remote_src": null,
            "removes": null,
            "return_content": false,
            "selevel": null,
            "serole": null,
            "setype": null,
            "seuser": null,
            "src": null,
            "status_code": [
                "200"
            ],
            "timeout": 30,
            "unsafe_writes": null,
            "url": "\"http\" \"://10.128.18.163\" \":8200/v1/sys/health\"\n",
            "url_password": null,
            "url_username": null,
            "use_proxy": true,
            "validate_certs": false
        }
    },
    "msg": "Status code was -1 and not [200]: Request failed: <urlopen error unknown url type: \"http\" \">",
    "redirected": false,
    "retries": 31,
    "status": -1,
    "url": "\"http\" \"://10.128.18.163\" \":8200/v1/sys/health\"\n"
}
```

**Comment by brianshumate** [comment]
Hi!
Sorry, I was trying a change in syntax to cope with long lines to help with YAML linting and picked the wrong method. I've since reverted that change and it will be shortly available in v2.1.7.

**Comment by brianshumate** [comment]
OK, this is fixed now in v2.1.7 of the role. I will go ahead and close this.
